README.md

```diff
@@ -100,21 +100,34 @@ conda activate koalas-dev-env
 
 ### Running Tests
 
+There is a script `./dev/pytest` which is exactly same as `pytest` but with some default settings to run Koalas tests easily.
+
 To run all the tests, similar to our CI pipeline:
 ```bash
-./dev/run-tests.sh
+# Run all unittest and doctest
+./dev/pytest
 ```
 
 To run a specific test file:
 ```bash
-python databricks/koalas/tests/test_dataframe.py
+# Run unittest
+./dev/pytest -k test_dataframe.py
```

Comment:
Strictly, there look other ways to specify as below:

```
pytest databricks/koalas/tests/test_dataframe.py
```

but I think it's easier for us just to stick to `-k` expression for both doctest and unittest. It's even shorter.

I also renamed `run-tests.sh` to `pytest` as it is exactly same as `pytest` but only have some configuration and/or default settings.

Reply:
I think in this way, we can just forget about documentation about how to test but we can just redirect the details into `pytest`.